dist/info/atari800_libretro.info

```diff
@@ -0,0 +1,17 @@
+display_name = "Atari 5200 (Atari800)"
+authors = "Petr Stehlik|r-type"
+supported_extensions = "a52"
+corename = "Atari800"
```

Comment:
What core is this? I don't see it in https://wiki.libretro.com/index.php?title=Category:Libretro_Cores .